README.md

```diff
@@ -73,19 +73,18 @@ using a git repository and
 1.  Ensure that you either have **Owner** permissions for the whole Cloud
     project, or at minimum the **Monitoring Editor** role
 1.  Create a ts-bridge config with no metrics
-    *   `cd ~/gopath/src/github.com/google/ts-bridge/app; cp metrics.yaml.example metrics.yaml`
+    *   `cd ~/gopath/src/github.com/google/ts-bridge/app; cp
+        metrics.yaml.example metrics.yaml`
     *   Edit the yaml file, remove the datadog\_metrics sample content, and copy
         in the name of the project you just created into the
         stackdriver\_destinations section.
-    *   Your `metrics.yaml` file should look like this:
-    ```
-    datadog_metrics:
-    stackdriver_destinations:
-      - name: stackdriver
-        project_id: "your_project_name"
-    ```
-1.  Turn on the status page (uncomment #ENABLE\_STATUS\_PAGE: "yes" in `app.yaml`)
-1.  Update `SD_PROJECT_FOR_INTERNAL_METRICS` in your `app.yaml` to match the name of your GCP project.
+    *   Your `metrics.yaml` file should look like this: `datadog_metrics:
```

Comment:
This borks the formatting, turning the multiline config file example into a single line. I suggest you separate content changes from formatting changes to make review of new content easier. As it is I'm having trouble finding the new content.

Reply:
Done.